From 8339d24b4553dd59c382f5f533516e57e901a808 Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Thu, 20 Mar 2025 19:05:33 +0100
Subject: [PATCH 809/954] feat(st): Update metadata and fip offset for nor

Adapt the NOR offset to the new layout.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 6e499e538ee7778530594bf71df12715a884ea2a feat(st): Update metadata and fip offset for nor]
Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
Change-Id: I97e6104e57c2197de4466132fcc21a62751a2589
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/445621
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
Tested-by: Maxime MERE <maxime.mere@foss.st.com>
Domain-Review: Maxime MERE <maxime.mere@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 plat/st/common/include/stm32mp_io_storage.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/plat/st/common/include/stm32mp_io_storage.h b/plat/st/common/include/stm32mp_io_storage.h
index fb7c2d658a..7e0194c24d 100644
--- a/plat/st/common/include/stm32mp_io_storage.h
+++ b/plat/st/common/include/stm32mp_io_storage.h
@@ -17,10 +17,10 @@
 #if PSA_FWU_SUPPORT
 #if STM32MP_SPI_NOR
 #if defined(STM32MP_M33_TDCID) && STM32MP_M33_TDCID
-#define STM32MP_NOR_METADATA1_OFFSET		U(0x01580000)
-#define STM32MP_NOR_METADATA2_OFFSET		U(0x015C0000)
-#define STM32MP_NOR_FIP_A_OFFSET		U(0x01600000)
-#define STM32MP_NOR_FIP_B_OFFSET		U(0x01A00000)
+#define STM32MP_NOR_METADATA1_OFFSET		U(0x01480000)
+#define STM32MP_NOR_METADATA2_OFFSET		U(0x014C0000)
+#define STM32MP_NOR_FIP_A_OFFSET		U(0x01500000)
+#define STM32MP_NOR_FIP_B_OFFSET		U(0x01900000)
 #else /* STM32MP_M33_TDCID */
 #define STM32MP_NOR_METADATA1_OFFSET		U(0x00080000)
 #define STM32MP_NOR_METADATA2_OFFSET		U(0x000C0000)
-- 
2.53.0

